api/fetchers/rpms_images_fetcher.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from ghapi.all import GhApi
+
+GITHUB_TOKEN = os.environ.get("GITHUB_PERSONAL_ACCESS_TOKEN")
+
+REPO_OWNER = "openshift-eng"
+REPO_NAME = "ocp-build-data"
+
+api = GhApi(owner=REPO_OWNER, repo=REPO_NAME, token=GITHUB_TOKEN)
+
+
+def get_directory_contents(branch, directory):
+    return api.repos.get_content(owner=REPO_OWNER, repo=REPO_NAME, path=directory, ref=branch)
+
+
+def fetch_data(release):
+    result = []
+    branches = api.list_branches()
+
+    for branch in branches:
```

Comment:
we don't need to loop the branches, we can directly check the branch for release